introduction.md

```diff
@@ -1,2 +1,6 @@
 An introduction as requested ;-)
-Favorite path: Ruby, Git, Javascript.
\ No newline at end of file
+
+Favorite path:
+* Ruby
+* Git
+* Javascript.
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?